end_to_end/tpu/llama2/7b/test_llama2_7b.sh

```diff
@@ -53,10 +54,11 @@ python3 -m MaxText.decode MaxText/configs/base.yml load_parameters_path=${UNSCAN
 python3 -m MaxText.decode MaxText/configs/base.yml load_parameters_path=${CONVERTED_CHECKPOINT} run_name=runner_direct_${idx} base_output_directory=${BASE_OUTPUT_DIRECTORY} per_device_batch_size=1 model_name='llama2-7b' ici_autoregressive_parallelism=4 max_prefill_predict_length=4  max_target_length=16 prompt="I love to" attention=dot_product
 
 # Alternatively, we skip to running finetuning by using the scanned converted checkpoint located at `CONVERTED_CHECKPOINT`. Again, we use it by specifying`load_parameters_path=${CONVERTED_CHECKPOINT}`. Note that scanned checkpoint helps with efficient finetuning
-python3 -m MaxText.train MaxText/configs/base.yml load_parameters_path=${CONVERTED_CHECKPOINT} run_name=runner_finetuning_${idx} base_output_directory=${BASE_OUTPUT_DIRECTORY} dataset_path=${DATASET_PATH} async_checkpointing=${ASYNC_CHECKPOINTING} per_device_batch_size=1 model_name='llama2-7b' ici_tensor_parallelism=4 steps=10 max_target_length=1024 per_device_batch_size=1 checkpoint_period=5
-
+# We run this with context parallelism and the `ici_context_parallelism` flag as an integration test for context parallelism
```

Comment:
I think in a very short time llama2 tests will go away in favor of llama3.1. Can we have this new logic as part of a llama3.1 test ?

Reply:
Added it in `end_to_end/tpu/llama3.1/8b/2_test_llama3.1_8b.sh`, I will let is in this llama2 file too since this is the one which is run nightly